packages/react/src/__tests__/ReactElementValidator-test.js

```diff
@@ -434,6 +434,29 @@ describe('ReactElementValidator', () => {
     );
   });
 
+  it('should warn if component declares PropTypes instead of propTypes', () => {
+    spyOn(console, 'error');
+    class MisspelledPropTypesComponent extends React.Component {
+      static PropTypes = {
+        prop: PropTypes.string,
+      };
+      render() {
+        return React.createElement('span', null, this.props.prop);
+      }
+    }
+
+    ReactTestUtils.renderIntoDocument(
+      React.createElement(MisspelledPropTypesComponent, {prop: 'Hi'}),
+    );
+
+    expectDev(console.error.calls.count()).toBe(1);
+    /*eslint-disable max-len */
```

Comment:
Nit: instead of disabling, you can split the message into multiple lines and concatenate them. Other tests do this.

Reply:
ok i removed eslint disable and added the split in string.